Events/Ch14.event

```diff
@@ -59,10 +56,11 @@ UNIT Eirika EirikaLord 0 Level(1, Ally, 0) [6,2] 0 0 0 0 [0] NoAI
 UNIT
 
 Ch14_Enemies:
-UNIT Wicky Witch 0 Level(10, Enemy, 0) [6,18] 0 0 0 0 [Luna, Elfire] GuardTileAI
-UNIT Ch14_Enemy Bishop 0 Level(Ch14_Level2, Enemy, 1) [3,10] 0 0 0 0 [Physic] GuardTileAI
-UNIT Ch14_Enemy Gargoyle 0 Level(Ch14_Level1, Enemy, 1) [1,18] 0 0 0 0 [SteelLance] NoAI
+//UNIT Wicky Witch 0 Level(10, Enemy, 0) [6,18] 0 0 0 0 [Luna, Elfire] GuardTileAI
+//UNIT Ch14_Enemy Bishop 0 Level(Ch14_Level2, Enemy, 1) [3,10] 0 0 0 0 [Physic] GuardTileAI
+//UNIT Ch14_Enemy Gargoyle 0 Level(Ch14_Level1, Enemy, 1) [1,18] 0 0 0 0 [SteelLance] NoAI
```

Comment:
Why are these now commented out? They worked before. If you intend to remove them, please remove them.